Resources/Prototypes/Entities/Objects/Weapons/Melee/e_sword.yml

```diff
@@ -317,3 +317,12 @@
       right:
       - state: inhand-right-blade
         shader: unshaded
+
+- type: entity
+  name: borg double-bladed energy sword # could improve name
```

Comment:
don't change the name to borg blade, keep it the same. you might be able to delete this line and it might inherit it from its parent, EnergySwordDouble.

add "suffix: One-Handed, For Borgs" or something of that nature so you can tell the two entities apart in the spawn menu

Reply:
also screenshot the two deswords in the spawn menu for me after you add the suffix, I suffer from a strong case of skill issue and have no clue if this works

Reply:
Image of the two side-by-side with new inherited name and suffix (both inspected with the magic of MS paint)
![image](https://github.com/user-attachments/assets/41bb60b2-9b59-4cf6-9f23-b24df7f64bd2)

Image of the two DESwords in spawn menu
![image](https://github.com/user-attachments/assets/23a55979-46e3-43b1-9b4d-66dfb631fe5d)

Reply:
thanks